### PR TITLE
Refactor: Combine dependencies & versions into a single file

### PR DESCRIPTION
**Pull request by gizemb**
### Issues

It is hard to track dependencies in separate modules. We combined all into a single file to be able to search and update them from a single place.

### Notes

Shouldn't change any behavior since there's no version update.

### CODE REVIEW

File: dependencies.gradle [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        kluent      : "1.14",
+        lifecycle   : "2.0.0",
+        okHttp      : "3.12.0",
+        circe       : "0.9.3",
+        retrofit    : "2.6.2",
+        macwire     : "2.3.3",
+        glide       : "4.10.0"
+    ]
+
+    deps = [
```
Comment (BradleyWire) [inline dependencies.gradle]: Would we not want more key dependencies in this collection/ Maybe at-least the ones shared across multiple modules?

Comment (android10) [reply dependencies.gradle]: This file In general is growing in size, making it less and less readable and hard to understand. We will need to extract many of this stuff to separated files. AFAIK dependencies could sit in a ```dependencies.gradle``` that can be merged with the main ```gradle``` file

**Comment by android10** [comment]
@gizemb what is the current state of this one?

**Comment by android10** [comment]
@gizemb closing this one in favor of #2514. Great job in any case :+1: :tada: :)

**Pull request closed without merging**